src/index.html

```diff
@@ -6,6 +6,14 @@
     <title>Capstone project</title>
 </head>
 <body>
-    <h1>Hello you're welco</h1>
+    <header>
+         <h2>Paint-<span>art</span></h2>
```

Comment:
Your styles.css file uses a **header h1** selector. I think you meant to use an **h1** tag here. It also is more appropriate in this case since it is the main title of our page. Hence, can you please use an **h1** here?